src/plugins/js-eval/jsEvalPlugin.js

```diff
@@ -33,23 +29,20 @@ module.exports = async function jsEvalPlugin({
 
   const hasMaybeTLA = /\bawait\b/.test(code);
 
-  if (mode === 'b' && !hasMaybeTLA) {
-    code = (await babel.transformAsync(code, { plugins: transformPlugins })).code;
```

Comment:
done a fix, because that code above was not using parserOpts: {plugins: parserPlugins} before

Reply:
this should allow us to use records and tuples, or other babel syntax things, but I'm not totally sure